Generalize lemmas about `fDiv` (#38)
RemyDegenne/testing-lower-bounds: blueprint/src/sections/f_divergence.tex

  \lean{ProbabilityTheory.condFDiv_ne_top_iff_fDiv_compProd_ne_top}
  \leanok
  \uses{def:fDiv, def:condFDiv}

>   Let $\mu$ be a finite measure on $\mathcal X$ and let $\kappa, \eta : \mathcal X \rightsquigarrow \mathcal Y$ be two finite kernels, such that $\kappa (x) \neq 0 \; \forall x$.

Reviewer: Don't use $\forall$ or $\exists$ in a sentence.
```suggestion
  Let $\mu$ be a finite measure on $\mathcal X$ and let $\kappa, \eta : \mathcal X \rightsquigarrow \mathcal Y$ be two finite kernels, such that $\kappa(x) \ne 0$ for all $x$.
```

  \lean{ProbabilityTheory.fDiv_compProd_left}
  \leanok
  \uses{def:fDiv, def:condFDiv}

>   Let $\mu$ be a finite measure on $\mathcal X$ and let $\kappa, \eta : \mathcal X \rightsquigarrow \mathcal Y$ be two finite kernels, such that $\kappa (x) \neq 0 \; \forall x$.

Reviewer: ```suggestion
  Let $\mu$ be a finite measure on $\mathcal X$ and let $\kappa, \eta : \mathcal X \rightsquigarrow \mathcal Y$ be two finite kernels, such that $\kappa(x) \ne 0$ for all $x$.
```

  \lean{ProbabilityTheory.fDiv_comp_left_le}
  \leanok
  \uses{def:fDiv, def:condFDiv}

>   Let $\mu$ be a measure on a standard Borel space $\mathcal X$ and let $\kappa, \eta : \mathcal X \rightsquigarrow \mathcal Y$ be two finite kernels, such that $\kappa (x) \neq 0 \; \forall x$.

Reviewer: ```suggestion
  Let $\mu$ be a measure on a standard Borel space $\mathcal X$ and let $\kappa, \eta : \mathcal X \rightsquigarrow \mathcal Y$ be two finite kernels, such that $\kappa(x) \ne 0$ for all $x$.
```
